### PR TITLE
Add an `up_to_date` state to the `win_wua` state module

### PR DESCRIPTION
**Pull request by twangboy**
### What does this PR do?
Adds an `wua.up_to_date` state that will apply all Windows Updates.
Fixes a documentation error on the `win_wua` execution module

### What issues does this PR fix or reference?
https://github.com/saltstack/salt/issues/43272

### Previous Behavior
You could use `wua.installed` to install specific updates, but not all updates matching certain criteria.

### New Behavior
You can now use `wua.up_to_date` to install all updates matching certain criteria

### Tests written?
No

### CODE REVIEW

**Comment by lorengordon** [comment]
How about naming it `wua.uptodate`, for consistency with the [pkg state](https://docs.saltstack.com/en/latest/ref/states/all/salt.states.pkg.html#salt.states.pkg.uptodate)?

**Comment by twangboy** [comment]
I deliberated on that... The current naming convention is to separate words by underscores... and since it is not a package manager I decided to use the underscores.

**Comment by lorengordon** [comment]
Your call. The lack of a more structured grammar for salt means users are constantly surprised by subtle differences like this between and even within modules. Salt's great, don't get me wrong. But consistency goes a long way to building a good user experience.

**Comment by cachedout** [comment]
Certainly, it's your call but IMHO @lorengordon makes an excellent point about consistency here.

**Comment by twangboy** [comment]
@lorengordon @cachedout I have changed it to `uptodate`... and rebased...

File: salt/modules/win_wua.py [hunk 1]
```diff
@@ -110,7 +110,7 @@ def available(software=True,
             Include software updates in the results (default is True)
 
         drivers (bool):
-            Include driver updates in the results (default is False)
+            Include driver updates in the results (default is True)
```
Comment (terminalmage) [inline salt/modules/win_wua.py]: It looks like this was always wrong, the result of a copypasta when this function was added. I've opened https://github.com/saltstack/salt/pull/43742 to fix this in the 2017.7 branch as well.

**Review by terminalmage — APPROVED** [review]

**Pull request merged**